sympy/algebras/quaternion.py

```diff
@@ -363,18 +374,18 @@ def pow(self, p):
 
         Parameters
         ==========
-
-        p : number
-            The power to which current (self) quaternion is raised.
+        
```

Comment:
Failing because of trailing whitespaces. Please take care of this in future.

Reply:
Thanks again 😃